Lib/test/test_gc.py

```diff
@@ -1039,6 +1039,49 @@ class Z:
         gc.enable()
 
 
+# These tests need to be run in a separate process since gc.immortalize_heap
+# will mess up with the reference count of other tests
+class GCImmortalizeTests(unittest.TestCase):
```

Comment:
For these tests, after we conditionally expose the gc functions, we can add the skip decorator to skip them if said functions are not present.

Reply:
Makes sense, I'll update it accordingly!